R/approved_list.R

```diff
@@ -28,13 +28,20 @@ create_approved_list <- function(new_approved_list){
 
     # Merge existing approved_lists with new approved_lists
     Brood_approved_list <- dplyr::bind_rows(approved_list$Brood_approved_list,
-                                            new_approved_list$Brood_approved_list)
+                                            new_approved_list$Brood_approved_list) %>%
+      tidyr::drop_na()
+
     Capture_approved_list <- dplyr::bind_rows(approved_list$Capture_approved_list,
-                                              new_approved_list$Capture_approved_list)
+                                              new_approved_list$Capture_approved_list) %>%
+      tidyr::drop_na()
+
     Individual_approved_list <- dplyr::bind_rows(approved_list$Individual_approved_list,
-                                                 new_approved_list$Individual_approved_list)
+                                                 new_approved_list$Individual_approved_list) %>%
+      tidyr::drop_na()
+
     Location_approved_list <- dplyr::bind_rows(approved_list$Location_approved_list,
-                                               new_approved_list$Location_approved_list)
+                                               new_approved_list$Location_approved_list) %>%
+      tidyr::drop_na()
```

Comment:
Do we expect there to be NAs? If we use `drop_na` with no column specified it will drop any row where there is at least one NA, so we should use it with caution (or specify the columns where we NAs are unacceptable).

Reply:
This is added because the dummy_approved_list contains full-NA rows. Do you see an alternative way to deal with that?

Reply:
I've tested this again and I was wrong: it is not needed at all. So I removed the ```drop_na```.